apps/bridge/src/utils/hooks/usePrepareFinalizeCCTPBridge.ts

```diff
@@ -0,0 +1,44 @@
+import { usePrepareContractWrite } from 'wagmi';
+import getConfig from 'next/config';
+import MessageTransmitter from 'apps/bridge/src/contract-abis/MessageTransmitter';
+
+const { publicRuntimeConfig } = getConfig();
+
+const BRIDGE_DIRECTION_TO_MESSAGE_TRANSMITTER: Record<'deposit' | 'withdraw', `0x${string}`> = {
```

Comment:
I like the way U've set this up here, good naming and use of data structures to make intent clear